SoftLayer/CLI/virt/usage.py

```diff
@@ -0,0 +1,45 @@
+"""Usage information of a virtual server."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+from SoftLayer.CLI import helpers
+
+
+@click.command(short_help="Usage information of a virtual server.")
+@click.argument('identifier')
+@click.option('--start_date', '-s', type=click.STRING, required=True, help="Start Date e.g. 2019-3-4")
+@click.option('--end_date', '-e', type=click.STRING, required=True, help="End Date e.g. 2019-4-2")
+@click.option('--valid_type', '-t', type=click.STRING, required=True,
+              help="Metric_Data_Type keyName e.g. CPU0, CPU1, MEMORY_USAGE, etc.")
+@click.option('--summary_period', '-summary', type=click.INT, required=True,
```

Comment:
`-summary` should be `-p` or something similar. instead of having this be required, have it set to a default value of 600.

Reply:
Done.

The usage vs will show the average information too, it is for that I am using summary period by default 1800 to match with the control portal result.